clinica/pipelines/t1_linear/README.md

```diff
@@ -0,0 +1,70 @@
+# `t1-linear`` - <VERY_SHORT_DESCRIPTION>
```

Comment:
Can be deleted.

(I'm also wondering if we should remove this from `clinica generate` :thinking: )